backend/backend/Controllers/UsersController.cs

```diff
@@ -1,25 +1,45 @@
-﻿using backend.Models;
+﻿using backend.JwtAuthentication;
+using backend.Models;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace backend.Controllers
 {
     [Route("api/[controller]")]
+    [Authorize]
```

Comment:
could you explain this annotation?

Reply:
Means that only authenticated users can access these endpoint

Except the authentication endpoint as it has AllowAnonymous attribute